apps/pie-docs/src/assets/styles/components/_contentImage.scss

```diff
@@ -60,6 +59,16 @@ $contentImage-height-l--mobile:           400px;
     }
 }
 
+.c-contentImage--hasBackdrop .c-contentImage-backdrop,
+.c-contentImage--hasPadding {
+    background-color: f.$color-container-subtle;
+}
+
+.c-contentImage--hasPadding {
```

Comment:
is hasPadding the most descriptive class name if it also includes rounded borders?

Reply:
Any suggestions? Not really sure what else to name it. It's just this part.
`backdropHasPadding`? `gapBetweenImageAndBackdrop`?
![Screenshot 2023-11-30 at 11 24 37](https://github.com/justeattakeaway/pie/assets/24740015/e0427167-a317-4013-8880-40fdc4349882)

Reply:
off the top of my head `--rounded` and `--paddedBackdrop` or something (its a bit hard to tell which part of that image you mean - is the gap the pale grey space surrounding the image?

Reply:
alternatively, leave hasPadding but move radius to `--rounded`